### ISSUE TITLE
Not working for me in Borderlands 3

### ISSUE THREAD

**Issue by Isron-C** [z0z0z/mf-install#63]
Hi, thanks for your work.
I tried to use the script as it's describe but i have an "error" i think it's for that the "TV screen" stay grey in the game. I have tried only one of those two script like it's mentioned in an other post but nothing..
"wine: /home/isron/.steam/steam/steamapps/compatdata/397540/pfx is not owned by you"
I'm the only user and i enter the command as root.
I have tried to move "397540" in a other folder and run the game for generate a new but nothing too..

**Comment by z0z0z** [comment]
Try not running as root.

**Comment by Isron-C** [comment]
Shame, ok thank you, it's good now, no error again. 
But nothing in game, the screen stay grey.
I have tried running mf-install alone, install-mf-64 alone and those 2 (with mfplat) but nothing.. 
Idea? A mistake again ?

**Comment by z0z0z** [comment]
Try moving/deleting the Wine prefix, running it once to generate a new one, and then running ONE workaround script, just ONE.

**Comment by Isron-C** [comment]
Yes i tried it like you said in #57 but if i move the folder "397540", the game was like 'corrupt' and he want not start and i had a error message. Just have to uninstall the game and download it 
again.. 
I have always start the game one time for create the folder like you say in #57 
I did this 3 times, run mf-install alone -> nothing -> bug when i change de folder -> uninstall. 
mf-installcab alone -> nothing -> bug again when i change de folder -> uninstall. 
I don't know why
